setup.py

```diff
@@ -78,6 +84,23 @@
             'alldeps': (
                 'numpy >= {}'.format(NUMPY_MIN_VERSION),
                 'scipy >= {}'.format(SCIPY_MIN_VERSION),
+                'matplotlib>={}'.format(MATPLOTLIB_MIN_VERSION),
+                'scikit-image>={}'.format(SCIKIT_IMAGE_MIN_VERSION),
+                'pandas>={}'.format(PANDAS_MIN_VERSION),
+                'seaborn>={}'.format(SEABORN_MIN_VERSION),
+                'pytest>={}'.format(PYTEST_MIN_VERSION),
+                'pytest-cov>=2.9.0',
+                'Cython>=0.29.19',
+                'flake8>=3.8.2',
+                'mypy>=0.770',
+                'coverage>=5.1',
+                'sphinx>=2.1.2',
+                'sphinx-gallery>=0.7.0',
+                'Pillow>=7.1.2',
+                'joblib>=0.15.1',
+                'memory_profiler>=0.57.0',
+                'packaging>=20.4',
+                'numpydoc>=0.9.2'
```

Comment:
why not using format on it and here yes :
                'numpy >= {}'.format(NUMPY_MIN_VERSION),
                 'scipy >= {}'.format(SCIPY_MIN_VERSION),
                 'matplotlib>={}'.format(MATPLOTLIB_MIN_VERSION),
                 'scikit-image>={}'.format(SCIKIT_IMAGE_MIN_VERSION),
                 'pandas>={}'.format(PANDAS_MIN_VERSION),
                 'seaborn>={}'.format(SEABORN_MIN_VERSION),
                 'pytest>={}'.format(PYTEST_MIN_VERSION),